miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniAppDownloader.kt

```diff
@@ -22,11 +23,9 @@ internal class MiniAppDownloader(
     // Only run the latest version of specified MiniApp.
     suspend fun getMiniApp(appId: String, versionId: String): String = when {
         !isLatestVersion(appId, versionId) -> throw sdkExceptionForInvalidVersion()
-        miniAppStatus
-            .isVersionDownloaded(appId, versionId) -> storage.getMiniAppVersionPath(
-            appId,
-            versionId
-        )
+        File(storage.getMiniAppVersionPath(appId, versionId)).exists()
```

Comment:
Couldn't we move this file check into the `MiniAppStatus` class?

Reply:
Oh, you updated while I was looking at it 😆

Reply:
Sure. I'll move it into MiniAppStatus.